tutorials/dev-app-cli.adoc

```diff
@@ -0,0 +1,162 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dev-app-cli"]
+= Tutorial: Deploying an application by using the CLI
+include::_attributes/common-attributes.adoc[]
+:context: dev-app-cli
+
+toc::[]
+
+This tutorial walks you through deploying services to stand up an application called `national-parks-app` on {product-title} that displays a map of national parks across the world. You will use the {oc-first} to complete this tutorial.
+
+To complete this tutorial, you will perform the following steps:
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-creating-new-project_dev-app-cli[Create a new project for the application].
++
+This step allows your application to be isolated from other cluster user's workloads.
```

Comment:
```suggestion
Create a project so that your application can be isolated from other cluster user's workloads.
```
Would it be incorrect/weird/wrong to say "...can be isolated from other people's workloads."?